src/delimiter.js

```diff
@@ -440,34 +455,39 @@ const makeSqrtImage = function(
 
     if (delim.type === "small") {
         // Get an SVG that is derived from glyph U+221A in font KaTeX-Main.
-        viewBoxHeight = 1000 + vbPad;  // 1000 unit glyph height.
+        // 1000 unit normal glyph height.
+        viewBoxHeight = 1000 + 1000 * extraViniculum + vbPad;
         if (height < 1.0) {
             sizeMultiplier = 1.0;   // mimic a \textfont radical
         } else if (height < 1.4) {
             sizeMultiplier = 0.7;   // mimic a \scriptfont radical
         }
-        spanHeight = (1.0 + emPad) / sizeMultiplier;
-        texHeight = 1.00 / sizeMultiplier;
-        span = sqrtSvg("sqrtMain", spanHeight, viewBoxHeight, options);
+        spanHeight = (1.0 + extraViniculum + emPad) / sizeMultiplier;
+        texHeight = (1.00 + extraViniculum) / sizeMultiplier;
+        span = sqrtSvg("sqrtMain", spanHeight, viewBoxHeight, extraViniculum,
+            options);
         span.style.minWidth = "0.853em";
         advanceWidth = 0.833 / sizeMultiplier;  // from the font.
 
     } else if (delim.type === "large") {
         // These SVGs come from fonts: KaTeX_Size1, _Size2, etc.
         viewBoxHeight = (1000 + vbPad) * sizeToMaxHeight[delim.size];
-        texHeight = sizeToMaxHeight[delim.size] / sizeMultiplier;
-        spanHeight = (sizeToMaxHeight[delim.size] + emPad) / sizeMultiplier;
-        span = sqrtSvg("sqrtSize" + delim.size, spanHeight, viewBoxHeight, options);
+        texHeight = (sizeToMaxHeight[delim.size] + extraViniculum) / sizeMultiplier;
+        spanHeight = (sizeToMaxHeight[delim.size] + extraViniculum + emPad)
+            / sizeMultiplier;
+        span = sqrtSvg("sqrtSize" + delim.size, spanHeight, viewBoxHeight,
+            extraViniculum, options);
         span.style.minWidth = "1.02em";
         advanceWidth = 1.0 / sizeMultiplier; // 1.0 from the font.
 
     } else {
         // Tall sqrt. In TeX, this would be stacked using multiple glyphs.
         // We'll use a single SVG to accomplish the same thing.
-        spanHeight = height + emPad;
-        texHeight = height;
-        viewBoxHeight = Math.floor(1000 * height) + vbPad;
-        span = sqrtSvg("sqrtTall", spanHeight, viewBoxHeight, options);
+        spanHeight = height + extraViniculum + emPad;
+        texHeight = height + extraViniculum;
+        viewBoxHeight = Math.floor(1000 * height + extraViniculum) + vbPad;
+        span = sqrtSvg("sqrtTall", spanHeight, viewBoxHeight, extraViniculum,
+            options);
         span.style.minWidth = "0.742em";
         advanceWidth = 1.056;
```

Comment:
Why don't we divide by `sizeMultiplier` here like in the other cases?

Reply:
Because LaTeX apparently does the same thing. In quicklatex.com, if you render two tall \sqrts side by side, with one of them set to `\scriptscriptstyle`, it's easy to see that the line weights are thinner in the `\scriptscriptstyle` radical. That tells us that the line weights are scaling with the style, and that tells us not to correct this expression by dividing by `sizeMultiplier`.